gridstatus/caiso.py

```diff
@@ -204,12 +206,17 @@ def get_load_forecast(self, date, end=None, sleep=4, verbose=False):
 
         df = df.sort_values("Time")
 
-        # todo - what is the actual time of the forecast?
-        df["Forecast Time"] = df["Time"].iloc[0]
+        df = self._add_forecast_publish_time(
+            df,
+            current_time,
+            # DAM Hourly Demand Forecast is published at 9:10 AM according to OASIS.
+            # ATLAS Reference > Publications > OASIS Publications Schedule
+            publish_time_offset_from_day_start=pd.Timedelta(hours=9, minutes=10),
+        )
 
         df = df[
             [
-                "Forecast Time",
+                "Publish Time",
```

Comment:
i think we generally put publish time column after the interval columns in other places. i know it was already like this here, but wondering if we should change while were in the function